NEWS

```diff
@@ -7,6 +7,7 @@ older versions.
 Current git version
 -------------------
 
+  * A child object 'focused_client' for each tag object.
```

Comment:
Maybe add a verb somewhere in that sentence?

Reply:
I've replaced 'A' by 'New', but added no verb because this is essentially how the items in the NEWS file were written so far: https://herbstluftwm.org/news.html (I think it makes sense, because otherwise the verbs will become very repetitive)

Reply:
Thanks! Makes sense and fits better into the pattern of the other entries